kml/serdes.cpp

```diff
@@ -468,28 +470,43 @@ void SaveTrackGeometry(Writer & writer, MultiGeometry const & geom)
   if (sz > 1)
   {
     linesIndent = kIndent8;
-    writer << kIndent4 << "<MultiGeometry>\n";
+    writer << kIndent4 << "<gx:MultiTrack>\n";
+    /// @TODO(KK): add the <altitudeMode>absolute</altitudeMode> if needed
   }
 
-  for (auto const & e : geom.m_lines)
+  for (size_t lineIndex = 0; lineIndex < geom.m_lines.size(); ++lineIndex)
   {
-    if (e.empty())
+    auto const & line = geom.m_lines[lineIndex];
+    if (line.empty())
     {
       ASSERT(false, ());
       continue;
     }
 
-    writer << linesIndent << "<LineString><coordinates>";
+    writer << linesIndent << "<gx:Track>\n";
```

Comment:
Should the old approach be used for tracks without timestamps, and gx:Track/MultiTrack for timestamped data?

Reply:
The track can successfully be saved using only the gx:track tags without the timestamps (only the points). 
Maybe only to support by the others apps (I dunno which)?